draft-ietf-mops-streaming-opcons.md

```diff
@@ -716,45 +717,56 @@ To that effect, the Consumer Technology Association (CTA), who owns the Web Appl
 
 It is worth noting explicitly that more complex network protocol stacks are certainly possible - for instance, packets with this protocol stack may be carried in a tunnel, or in a VPN. If these environments are present, streaming media operators may need to analyze their effects on applications as well.
 
-## UDP and Its Behavior {#udp-behavior}
+## Media Transport Over Reliable Transport Protocols {#tcp-behavior}
 
-Because UDP does not provide any feedback mechanism to senders to help limit impacts on other users, UDP-based application-level protocols have been responsible for the decisions that TCP-based applications have delegated to TCP - what to send, how much to send, and when to send it. Because UDP itself has no transport-layer feedback mechanisms, UDP-based applications that send and receive substantial amounts of information are expected to provide their own feedback mechanisms, and to respond to the feedback the application receives. This expectation is most recently codified in Best Current Practice {{RFC8085}}.
+The HLS {{RFC8216}} and DASH {{MPEG-DASH}} media transport protocols are typically carried over HTTP, and HTTP has used TCP as its only standardized transport protocol until recently. These media transport protocols use ABR response strategies as described in {{sec-abr}} to respond to changing path characteristics, and underlying transport protocols are also attempting to respond to changing path characteristics.
+
+The past success of the largely TCP-based Internet is evidence that the various flow control and congestion control mechanisms TCP has used to achieve equilibrium quickly, at a point where TCP senders do not interfere with other TCP senders for sustained periods of time ({{RFC5681}}), have been largely successful. The Internet has continued to work even when the specific TCP mechanisms used to reach equilibrium changed over time ({{RFC7414}}). Because TCP provided a common tool to avoid contention, even when significant TCP-based applications like FTP were largely replaced by other significant TCP-based applications like HTTP, the transport behavior remained safe for the Internet.
+
+Modern TCP implementations ({{I-D.ietf-tcpm-rfc793bis}}) continue to probe for available bandwidth, and "back off" when a network path is saturated, but may also work to avoid growing queues along network paths, which prevent TCP senders from detecting quickly when a network path becoming saturated. Congestion control mechanisms such as COPA {{COPA18}} and BBR {{I-D.cardwell-iccrg-bbr-congestion-control}} make these decisions based on measured path delays, assuming that if the measured path delay is increasing, the sender is injecting packets onto the network path faster than the network can forward them (or the receiver can accept them) so the sender should adjust its sending rate accordingly.
```

Comment:
nit: "when a network path becoming saturated": missing an "is"?  Or should this be "has become"?

But also, I'm not sure it's right that avoiding growing queues prevents TCP senders from detecting this quickly.  (For most bottlenecks BBR will level off at saturation quicker than reno will stop growing cwnd past it, for instance.)

Reply:
Thanks for asking this question - I meant **OLDER** TCP implementations, but didn't say that.